### PR TITLE
2 brokers and a load balancer

### PR DESCRIPTION
**Pull request by ilya-dubovskiy**

### CODE REVIEW

**Comment by ilya-martynov** [comment]
If you want to merge it you should add some reviewers.

File: docker/docker-compose/docker-compose.yaml [hunk 1]
```diff
@@ -1,63 +1,94 @@
 version: '2'
```
Comment (dmitriyisaev) [inline docker/docker-compose/docker-compose.yaml]: could we please create separate docker-compose for cluster solution. I mean we deffinetly should have 2 docker-compose files:
- for single mode
- for cluster mode.

Also would be better introduce flag in start shell script ro select mode.

File: docker/docker-compose/docker-compose-cluster.yaml [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+  broker-proxy:
+      image: eeacms/haproxy
+      depends_on:
+        - broker1
+        - broker2
+      volumes:
+        - /var/run/docker.sock:/var/run/docker.sock
+      ports:
+        - "80:5000"
```
Comment (c9alex) [inline docker/docker-compose/docker-compose-cluster.yaml]: Can you please change balancer port from '80' to '9080'?

**Review by c9alex — CHANGES_REQUESTED** [review]

**Review by dmitriyisaev — APPROVED** [review]
Looks fine

**Pull request closed without merging**